app/webpack/observations/uploader/actions/actions.js

```diff
@@ -149,7 +167,9 @@ const actions = class actions {
 
   static onFileDropOnCard( droppedFiles, obsCard ) {
     return function ( dispatch ) {
+      const cardFileCount = Object.keys( obsCard.files ).length;
```

Comment:
Here and below at 242 I think you need to make this check specific to photo files. Currently, if you have a card with 20 photos and you try to drag a sound on to it, it will give you the error message about only allowing 20 photos, which is confusing if you're adding a sound.

Reply:
Makes sense. Are you thinking 20 total files (photos and sounds) or 20 of each?

Reply:
20 of each.